### DISCUSSION TITLE
Trace to logs configuration

### DISCUSSION THREAD

**Discussion by andreivaloshyn** [grafana/tempo#1793] [Q&A] [answered]
Hello!
I need to configure trace to feature to be able to  observe logs related to trace(span) 
I found this article https://grafana.com/docs/grafana/latest/datasources/tempo/#trace-to-logs and it's not clear for me how it works.
Config for datasource:
<img width="796" alt="Screenshot 2022-10-11 at 22 24 12" src="https://user-images.githubusercontent.com/58686610/195191294-6b70610a-2b4c-4735-8b0d-f3701d00f234.png">
I'm able to see logs to trace
<img width="1376" alt="Screenshot 2022-10-11 at 22 25 03" src="https://user-images.githubusercontent.com/58686610/195191569-0fd22d92-5488-4db1-b1b3-a3065f0c8451.png">
but not vice versa 
<img width="1376" alt="Screenshot 2022-10-11 at 22 27 13" src="https://user-images.githubusercontent.com/58686610/195191868-7372f182-3c4a-436c-a173-f0b5c31000d0.png">

Tried on grafana  9.1.6 and 9.1.7.

What I did wrong or should add/update? :)

Thank you.

**Comment by joe-elliott** [comment, marked as answer]
When Grafana creates the trace -> logs link it uses the tags specified here:
![image](https://user-images.githubusercontent.com/2272392/195350259-280de2b1-e802-46b2-b273-0449a0abfd08.png)
to build the loki query. It's possible that it won't create the link unless you specify at least one tag. So if you have some tags in common between your traces and logs (things like cluster, namespace, pod) then you should add them to the field above.

Also going to ping @joey-grafana who works directly on these features.

**Reply by andreivaloshyn** [reply]
Great! Thank you, just matched tag and labels and it started to work.
